src/upasana/assignment_15/Bank.java

```diff
@@ -0,0 +1,169 @@
+/*
+ A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI
+ */
+
+package upasana.assignment_15;
+
+import java.util.Scanner;
+public class Bank {
+	int iciciBalT,hdfcBalT,bobBalT,sbiBalT;
+	int creditCountI=0,creditCountH=0,creditCountB=0,creditCountS=0;
+	int debitCountI=0,debitCountH=0,debitCountB=0,debitCountS=0;;
+	void operations(String op,String bank,int amt,int totAmt) {
+		switch(op){
+			case "debit":
+				switch(bank) {
+					case "ICICI":
+						iciciBalT=totAmt;
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method